### PR TITLE
[Bugfix] When user scroll up while open chat box, sometimes background is moving

### PR DESCRIPTION
**Pull request by ErickDoroy**
## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202252495766232/f
- Route Link: N/A

## Description

- When user scroll up while open chat box, sometimes background is moving

## Notes

N/A

## Screenshots

chatbot_bug

### CODE REVIEW

**Review by ume45876139 — APPROVED** [review]
# Working
## Chrome
<details>
<summary>PC</summary>

https://user-images.githubusercontent.com/39412730/167764394-62acbc59-9821-4d13-919a-9ebad296d712.mov


</details>

## Safari

<details>
<summary>Tablet</summary>


https://user-images.githubusercontent.com/39412730/167764560-6df87d05-5fbb-4b58-b32a-9004e37b2819.mp4



</details>

<details>
<summary>iPhone</summary>

https://user-images.githubusercontent.com/39412730/167764584-24de175f-c676-4ad0-abf1-934238d0c11d.MP4

</details>

- When user scroll up while open chat box, sometimes background is not moving
- About safari, if scroll is stopped completely, background is not moving. It seems Safari's spec



# Testing data
test device: Chrome, Safari 
test time: 0.75h
comment: 0
iteration: 0
total test time: 0.75h

**Review by ume45876139 — COMMENTED** [review]
miss

**Review by ume45876139 — APPROVED** [review]
# Working
## Chrome
<details>
<summary>Screenshot</summary>


https://user-images.githubusercontent.com/39412730/167785149-1b5a9d10-5600-4a4d-b353-2fdcf5db2a74.mov


</details>

<details>
<summary>Screenshot</summary>

https://user-images.githubusercontent.com/39412730/167785221-07507191-78e3-4455-ab08-795806d13797.mp4

</details>

<details>
<summary>Screenshot</summary>


https://user-images.githubusercontent.com/39412730/167785281-a6591ee7-a93e-4a6d-8c25-2aaab50de497.MP4


</details>

- PR is working



# Testing data
test device: Chrome, Safari (Mac)
test time: 0.25h
comment: 0
iteration: 0
total test time: 1h

File: src/pages/components/chatbot/widgets/Requirements.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { requirements } from '../../../../utilities/constants';
 
 const Requirements = () => {
   return (
-    <div className="border-2 p-4 pl-9 mx-12 bg-sky-100 rounded-lg">
-      <ul className=" list-disc">
+    <div className='border-2 p-4 pl-9 mx-12 bg-sky-100 rounded-lg'>
+      <ul className='list-disc'>
         {requirements.map((req) => {
           return (
             <li className="pb-3" key={req.id}>
```
Comment (egiegarciano14) [inline src/pages/components/chatbot/widgets/Requirements.js]:
```suggestion
            <li className='pb-3' key={req.id}>
```
change to single quote

File: src/pages/components/chatbot/widgets/Process.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { adoptionProcess } from '../../../../utilities/constants';
 
 const Process = () => {
   return (
-    <div className="border-2 p-5 pb-0 mx-12 bg-sky-100 rounded-lg">
+    <div className='border-2 p-5 pb-0 mx-12 bg-sky-100 rounded-lg'>
       <ul>
         {adoptionProcess.map((process) => {
           return (
```
Comment (egiegarciano14) [inline src/pages/components/chatbot/widgets/Process.js:9]: On line 11 and 12, change the quote to single

**Review by egiegarciano14 — APPROVED** [review]

**Pull request merged**